api/src/Repository/CampRepository.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Camp;
+use App\Entity\User;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\ORM\QueryBuilder;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @method null|Camp find($id, $lockMode = null, $lockVersion = null)
```

Comment:
Geht hier auch ?Camp, gehen hier auch php8 Attribute

Reply:
Weiss nicht ob das in PHPDoc akzeptiert ist, bei PHPSTORM geht damit jedenfalls das Syntax Highlighting verloren... php8 Attribute weiss ich auch nicht ob es das gibt, `@method` ist ja nur Dokumentation von Methoden die nicht unten implementiert stehen.

Reply:
ok, passt so